e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/keyboardNavigation.functional.ts

```diff
@@ -4629,3 +4629,62 @@ test('TreeList/DataGrid - Focus indicator is not visible when the Toolbar includ
   keyExpr: 'field_0',
   showBorders: true,
 }));
+
+test('Enter key should not trigger other function besides the function assigned on the clicked button', async (t) => {
```

Comment:
Let's add info about the SC ticket number 
There are two options how we can do it:

1) Add the SC ticket number to the test case name
```suggestion
test('Enter key should not trigger other function besides the function assigned on the clicked button (T1240353)', async (t) => {
```
Or 
2) Add a comment with the ticket number on top:
```suggestion
// T1240353
test('Enter key should not trigger other function besides the function assigned on the clicked button', async (t) => {
```